cpp/log/cert.cc

```diff
@@ -271,22 +262,12 @@ string Cert::PrintSerialNumber() const {
 
 
 string Cert::PrintIssuerName() const {
-  if (!IsLoaded()) {
-    LOG(ERROR) << "Cert not loaded";
-    return string();
-  }
-
-  return PrintName(X509_get_issuer_name(x509_.get()));
+  return PrintName(X509_get_issuer_name(CHECK_NOTNULL(x509_.get())));
```

Comment:
I'm worried about all these checks as it may make it impossible for tools to print out a cert that's not valid.